src/components/TransactionList/TransactionList.tsx

```diff
@@ -101,9 +90,22 @@ const TransactionList = (): React.ReactElement => {
         setOrderBy(property);
     };
 
-    const createSortHandler = (property: any) => (event: any) => {
-        handleRequestSort(event, property);
-    };
+    const tablePaginationProps = {
+        rowsPerPageOptions: [15, 25, 50, 100, 250],
+        colSpan: 3,
+        count: blokchain.length,
+        rowsPerPage: rowsPerPage,
+        page: page,
+        SelectProps: {
+            inputProps: { 'aria-label': 'rows per page' },
+            native: true,
+        },
+        onChangePage: handleChangePage,
+        onChangeRowsPerPage: handleChangeRowsPerPage,
+        ActionsComponent: TransactionListPagination
+    }
+
+    const createSortHandler = (property: any) => (event: any) => handleRequestSort(event, property);
```

Comment:
it's a comment to 'timestampToDate' in the line 110 (can't add comment) - Alan wrote in my pr earlier that it can be shorter https://github.com/ScalaConsultants/Blockchain-Analytic-Dashboard/pull/2#discussion_r319017873